docs/usage/machine.md

```diff
@@ -1,41 +1,62 @@
 # Creating/Deleting machines (VM)
+<!-- TOC -->
 
+- [Creating/Deleting machines (VM)](#creatingdeleting-machines-vm)
+  - [Setting up your usage environment](#setting-up-your-usage-environment)
+  - [Important :warning](#important-warning)
+  - [Creating machine](#creating-machine)
+  - [Inspect status of machine](#inspect-status-of-machine)
+  - [Delete machine](#delete-machine)
+
+<!-- /TOC -->
 ## Setting up your usage environment
 
 * Follow the [steps described here](prerequisite.md)
 
-## Important :warning: 
-- Make sure that the `kubernetes/machine.yaml` points to the same class name as the `kubernetes/aws-machine-class.yaml`.
-- Similarily `kubernetes/aws-machine-class.yaml` secret name and namespace should be same as that mentioned in `kubernetes/aws-secret.yaml`
+## Important :warning
+
+> Make sure that the `kubernetes/machines_sets_deployments/machine.yaml` points to the same class name as the `kubernetes/machine_classes/aws-machine-class.yaml`.
```

Comment:
Maybe we could rename `kubernetes/machines_sets_deployments` to `kubernetes/machine_objects`? does that make it more meaningful?

Reply:
Yes it makes sense, I just thought that for ease of access we include the type of the objects in the name but it turned out to be a long one.

Reply:
Thanks for the changes.